src/main/scala/com/example/Attemp2.scala

```diff
@@ -0,0 +1,113 @@
+package com.example
+
+import com.dimafeng.testcontainers.PostgreSQLContainer
+import doobie.free.connection.ConnectionIO
+import doobie.util.transactor.Transactor
+import zio.Task
+import doobie.implicits._
+import doobie.util.update.Update
+import fs2.Stream
+import zio._
+import zio.console._
+
+
+object Attemp2 extends App {
+
+
+  val program: ZIO[Console, Throwable, Unit] = for {
+    container   <- ZIO(PostgreSQLContainer())
+    _           <- IO.effectTotal(container.start())
+    xa          =  DatabaseInterface.getTransactor(container)
+    dbInterface =  DatabaseInterface(xa)
+    _           <- dbInterface.createTable
+//    _           <- dbInterface.create(User(1,"Anton"))
+//    _           <- dbInterface.create(User(2,"Sergei"))
+//    _           <- dbInterface.create(User(3,"Ivan"))
+//    _           <- dbInterface.create(User(4,"John"))
+//    _           <- dbInterface.insertMany(List(User(1,"Anton"), User(2,"Sergei"), User(3,"Ivan"), User(4,"John")))
+
+//    _           = Stream.randomSeeded(1488).map(User(_, "Yo")).evalMap(user => dbInterface.create(user)).take(100).compile.drain
+//    qq           = Stream.randomSeeded(1488).map(User(_, "Yo"))
+//    _= Stream.eval(ZIO.effect(qq.map(user => dbInterface.create(user)))).compile.drain
+//    _ <- putStr(qq.mkString)
+
+    _           <- dbInterface.insertMany((1 to 100).toList.map(User(_, "Vasya")))
+
+    queue       <- dbInterface.getQueue(50)
+    queueData   <- queue.takeAll
+    _           <- putStrLn(queueData.mkString)
+
+//    queueData   <- queue.takeAll
+//    _           <- putStrLn(queueData.toString)
+
+
+
+  } yield()
+
+  override def run(args: List[String]): ZIO[Attemp2.Environment, Nothing, Int] = {
+    program.fold(_ => 1, _ => 0)
+  }
+
+}
+
+case class DatabaseInterface(tnx: Transactor[Task]) {
+  import zio.interop.catz._
+  import cats.implicits._
+
+
+  val createTable: Task[Unit] =
```

Comment:
All methods in this file should either have scaladoc or have restricted visibility (private, protected).